videomaker/block-templates/page.html

```diff
@@ -16,12 +16,12 @@
 			<!-- wp:post-terms {"term":"category","style":{"typography":{"fontSize":"var(--wp--custom--font-sizes--tiny)"}}} /-->
 		</div>
 		<!-- /wp:group -->
-		<!-- wp:group {"className":"post-tags-container"} -->
-		<div class="wp-block-group post-tags-container">
+		<!-- wp:group {"className":"post-tags-container","style":{"spacing":{"margin":{"top":"var(--wp--style--block-gap,1em)","bottom":"calc(2 * var(--wp--style--block-gap,1em))"}}}} -->
+		<div class="wp-block-group post-tags-container" style="margin-top:var(--wp--style--block-gap,1em);margin-bottom:calc(2 * var(--wp--style--block-gap,1em))">
 			<!-- wp:post-terms {"term": "post_tag","style":{"typography":{"fontSize":"var(--wp--custom--font-sizes--tiny)"}}} /-->
```

Comment:
I adjusted this because it was in here, but I don't think Pages support `post_terms` by default? Also I'm not sure why this appears above the featured image here, but below it for single posts. 🤷 

Might be worth re-assessing those points in a followup PR.